File: Server/pizzeria-infrastructure/pizzeria.interface/Models/AllItems.cs
using System.Collections.Generic;

namespace pizzeria_api.interfaces.Models
{
    public class AllItems
    {
        public List<Pizza> Pizzas { get; set; }
        public List<NonPizzaItem> NonPizzaItems { get; set; }
        public List<Ingredient> Ingredients { get; set; }
        public List<Cheese> Cheese { get; set; }
        public List<CrustSauce> CrustSauces { get; set; }
        public List<CrustSize> CrustSizes { get; set; }
        public List<Topping> Toppings { get; set; }
    }
}

File: pizzeria-web/src/app/models/AllItems.ts
import { Cheese } from "./Cheese";
import { CrustSauce } from "./CrustSauce";
import { CrustSize } from "./CrustSize";
import { Ingredient } from "./Ingredient";
import { NonPizzaItem } from "./NonPizzaItem";
import { Pizza, PizzaUi } from "./Pizza";
import { Topping } from "./Topping";

export class AllItems {
    Pizzas: Pizza[] = [];
    NonPizzaItems: NonPizzaItem[] = [];
    Ingredients: Ingredient[] = [];
    Cheese: Cheese[] = [];
    CrustSauces: CrustSauce[] = [];
    CrustSizes: CrustSize[] = [];
    Toppings: Topping[] = [];
}

export class AllSelectedItems {
    Pizzas: PizzaUi[] = [];
    NonPizzaItems: NonPizzaItem[] = [];
    TotalAmmount: number = 0.00;
}